openapi/openapi.yaml

```diff
@@ -0,0 +1,274 @@
+openapi: 3.1.0
+info:
+  title: OpenAPI standard for federal grant data
+  version: 0.0.1
```

Comment:
A major version of 0 implies we're reserving the right to make breaking changes within `v0.x.x`. There may be an argument for doing so intentionally as our starting point – "nothing prods like prod"; it's entirely possible that we'll learn things upon initial release that necessitate breaking changes.

Alternatively, we could just start at `1.0.0` and if we find that breaking changes are necessary, we commit to defining a new major version and rapidly deprecating the old, which would push us to define our strategy for rolling out new versions in a consumer-friendly manner. 

What do you think?

Reply:
I'm aligned with starting at `1.0.0` 👍